LDAR_Sim/src/methods/company.py

```diff
@@ -260,11 +260,12 @@ def candidates_to_watchlist(self):
         # This ensures that the leakiest sites are at the top of the dictionary
         # so proportional site followup can be performed without resorting everyday
         # within the flag_sites function.
-        site_wl = {k: v for k, v in sorted(
-            site_wl.items(),
-            key=lambda x: x[1]['site_measured_rate'],
-            reverse=True)
-        }
+        if self.config['follow_up']['sort']:
```

Comment:
Would we want to randomly shuffle the sites here when we don't sort/does that happen each day somewhere else in the code?

Reply:
The surveys are ordered based on the sequence in which they were originally conducted, and we want to maintain this order based on the surveyed site. Essentially, whichever site gets flagged first will be followed up first.